src/Asset/ContaoContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Contao\CoreBundle\Asset;
+
+use Symfony\Component\Asset\Context\ContextInterface;
+
+class ContaoContext implements ContextInterface
```

Comment:
Nothing should rely on the constants anymore. If you need to access `TL_*` it's better to find a different way because the constants will not be available in 5 anymore.

Reply:
The only other way is to read from `$GLOBALS` in the back end and from `global $objPage` in the front end, which does not seem less wrong than using the constants to me.

Reply:
Yeah, or we find a new way because apparently we will still need that feature in the future :-)

Reply:
Added ToDo no. 3 for this. We should probably inject the PageModel and "duplicate" the code from `Controller::setStaticUrls`.